docs/BR.md

```diff
@@ -1935,44 +1944,60 @@ If the Subordinate CA is not allowed to issue certificates with dNSNames, then t
 
 ### 7.1.6 Certificate policy object identifier
 
-#### 7.1.6.1. Reserved Certificate Policy Identifiers
 This section describes the content requirements for the Root CA, Subordinate CA, and Subscriber Certificates, as they relate to the identification of Certificate Policy.
 
-The following Certificate Policy identifiers are reserved for use by CAs as a means of asserting compliance with these Requirements as follows:
+#### 7.1.6.1. Reserved Certificate Policy Identifiers
 
-`{joint-iso-itu-t(2) international-organizations(23) ca-browser-forum(140) certificate-policies(1) baseline-requirements(2) domain-validated(1)} (2.23.140.1.2.1)`, if the Certificate complies with these Requirements but lacks Subject Identity Information that is verified in accordance with Section 3.2.2.1 or Section 3.2.3.
+The following Certificate Policy identifiers are reserved for use by CAs as an optional means of asserting that a Certificate complies with these Requirements.
```

Comment:
Should we clarify that these policy identifiers are meant for end-entity Certificates?

"...of asserting that an end-entity Certificate complies with..."

Reply:
No, because as discussed on the validation list and GitHub, and this text itself, they are not just that. They also assert the CA complies with these, by definition, since they can be used to issue such certificates.

Reply:
I didn't follow these discussions on GitHub nor the validation SC list. The fact that a CA Certificate asserts one or more of the OIDs that are specifically designed for end-entity certificates (for DV, OV, EV) means that these oids are likely to appear in end-entity certificates, and then, by definition, the CA must comply with the  corresponding Issuer requirements. However, the fact that a CA Certificate contains subject information with and organizationName field and asserts the "DV" policy OID, doesn't mean that it violates the BRs.

Is that clear enough in this ballot?

I'm sorry if this has been answered already.